### PR TITLE
Update control-access-from-unmanaged-devices.md

### PR DESCRIPTION
**Pull request by ToniSFrench**
The allow Limited, web-only access prevents the previews of files from appear in SharePoint/Yammer integration scenarios, such as Yammer web parts/embed, so we want to note that limitation for users. If you have any technical questions, please reach out to the PM, Tricia Lybrook.

I couldn't find a better place for the proposed note, although I realize it's not ideal to have two notes back-to-back. Thanks! -v-tosadd (Toni Saddler-French, Yammer Writer).

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/control-access-from-unmanaged-devices.md [hunk 1]
```diff
@@ -80,6 +80,9 @@ Limiting access allows users to remain productive while addressing the risk of a
 
 >[!NOTE]
 > If you limit access and edit a site from an unmanaged device, image web parts won't display images that that you upload to the site assets library or directly to the web part. To work around this issue, you can use this [SPList API](/previous-versions/office/sharepoint-server/mt796229(v%3Doffice.15)) to exempt the block download policy on the site assets library. This allows the web part to download images from the site assets library.
+
+>[!NOTE]
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/control-access-from-unmanaged-devices.md:84]:
```suggestion
> [!NOTE]
```

File: SharePoint/SharePointOnline/control-access-from-unmanaged-devices.md [hunk 2]
```diff
@@ -80,6 +80,9 @@ Limiting access allows users to remain productive while addressing the risk of a
 
 >[!NOTE]
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/control-access-from-unmanaged-devices.md:81]:
```suggestion
> [!NOTE]
```

File: SharePoint/SharePointOnline/control-access-from-unmanaged-devices.md [hunk 3]
```diff
@@ -80,6 +80,9 @@ Limiting access allows users to remain productive while addressing the risk of a
...
 > If you limit access and edit a site from an unmanaged device, image web parts won't display images that that you upload to the site assets library or directly to the web part. To work around this issue, you can use this [SPList API](/previous-versions/office/sharepoint-server/mt796229(v%3Doffice.15)) to exempt the block download policy on the site assets library. This allows the web part to download images from the site assets library.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/control-access-from-unmanaged-devices.md:82]:
```suggestion
> If you limit access and edit a site from an unmanaged device, image web parts won't display images that you upload to the site assets library or directly to the web part. To work around this issue, you can use this [SPList API](/previous-versions/office/sharepoint-server/mt796229(v%3Doffice.15)) to exempt the block download policy on the site assets library. This allows the web part to download images from the site assets library.
```

File: SharePoint/SharePointOnline/control-access-from-unmanaged-devices.md (hunk 1 above)
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/control-access-from-unmanaged-devices.md:84]:
```suggestion
> [!NOTE]
```

File: SharePoint/SharePointOnline/control-access-from-unmanaged-devices.md [hunk 4]
```diff
@@ -80,6 +80,9 @@ Limiting access allows users to remain productive while addressing the risk of a
...
+
+>[!NOTE]
+> When Access Control for Unmanaged Devices in SharePoint is set to **Allow limited, web-only access**, SharePoint files cannot be downloaded but they can be previewed. The previews of Office files work in SharePoint but the previews do not work in Microsoft Yammer.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/control-access-from-unmanaged-devices.md:85]:
```suggestion
> When Access Control for Unmanaged Devices in SharePoint is set to **Allow limited, web-only access**, SharePoint files cannot be downloaded but they can be previewed. The previews of Office files work in SharePoint but the previews do not work in Microsoft Yammer.
```

File: SharePoint/SharePointOnline/control-access-from-unmanaged-devices.md (hunk 3 above)
Comment (JohanFreelancer9) [reply SharePoint/SharePointOnline/control-access-from-unmanaged-devices.md:82]: Not related to this PR, but please also correct the typo in the first NOTE (double word).

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**